cli/tests/integration_tests.rs

```diff
@@ -2715,6 +2715,25 @@ console.log("finish");
     assert!(end - start < Duration::new(10, 0));
   }
 
+  #[test]
+  #[ignore] // Hangs on CI, passes locally.
+  fn broadcast_channel() {
```

Comment:
So this is a integration test instead of a JS unit test because it requires --ignore?

Reply:
Because it requires `--location`.

Reply:
Oh... don't we run unit tests with `--location`? If not, @bartlomieju any objections to doing that?

Reply:
We do:
```
--location=http://js-unit-tests/foo/bar
```

Reply:
Oh, I remember the other issue: the test hangs when run as a unit test because `deno test` tries to type-check the embedded script. I guess I'll move it to a separate file.